modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -31,4 +51,97 @@ export class Utils implements BaseUtils {
   isValidTransactionId(txId: string): boolean {
     throw new NotImplementedError('method not implemented');
   }
+
+  /**
+   * Returns an hex string of the given buffer
+   *
+   * @param {Buffer | Uint8Array} buffer - the buffer to be converted to hex
+   * @returns {string} - the hex value
+   */
+  toHex(buffer: Buffer | Uint8Array): string {
```

Comment:
This should be in the https://github.com/BitGo/BitGoJS/tree/master/modules/account-lib/src/utils